src/ch/imedias/rsccfx/view/PopOverHelper.java

```diff
@@ -0,0 +1,268 @@
+package ch.imedias.rsccfx.view;
+
+import ch.imedias.rsccfx.RsccApp;
+import ch.imedias.rsccfx.ViewController;
+import ch.imedias.rsccfx.model.Rscc;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.geometry.Insets;
+import javafx.geometry.Rectangle2D;
+import javafx.geometry.VPos;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import javafx.stage.Screen;
+import org.controlsfx.control.PopOver;
+import org.controlsfx.control.ToggleSwitch;
+
+
+/**
+ * @author Lukas Marchesi
+ * @date 17.04.2017.
+ */
+public class PopOverHelper {
```

Comment:
I think this class is way too big and has too many dependencies... I'll talk with you about it